tier1-icons.html

```diff
@@ -29,14 +29,14 @@
         <g id="arrow-toggle-up">
           <path d="M13.7 10.29l-4-4a1.012 1.012 0 0 0-1.418 0l-4 4A1 1 0 0 0 5.007 12H13a1 1 0 0 0 .7-1.71z"/>
         </g>
-        <g id="assignments">
+        <g id="assignments" mirror-rtl="true">
           <path d="M18 12v5a1 1 0 0 1-1 1H1a1 1 0 0 1-1-1v-5a1 1 0 0 1 2 0v4h14v-4a1 1 0 0 1 2 0z"/>
           <path d="M13.85 3.15l-2.99-3A.507.507 0 0 0 10.5 0H5.4A1.417 1.417 0 0 0 4 1.43v11.14A1.417 1.417 0 0 0 5.4 14h7.2a1.417 1.417 0 0 0 1.4-1.43V3.5a.47.47 0 0 0-.15-.35zM7 2h1a1 1 0 0 1 0 2H7a1 1 0 0 1 0-2zm4 10H7a1 1 0 0 1 0-2h4a1 1 0 0 1 0 2zm0-4H7a1 1 0 0 1 0-2h4a1 1 0 0 1 0 2z"/>
         </g>
         <g id="attach">
           <path d="M15.85 3.15l-2.99-3A.508.508 0 0 0 12.5 0h-1.05A3.9 3.9 0 0 1 12 2v9a4.03 4.03 0 0 1-4 4 4.012 4.012 0 0 1-4-4V4a1 1 0 0 1 2 0v7a2 2 0 0 0 4 0V2a2.006 2.006 0 0 0-2-2H3.4A1.417 1.417 0 0 0 2 1.43v15.14A1.417 1.417 0 0 0 3.4 18h11.2a1.417 1.417 0 0 0 1.4-1.43V3.5a.47.47 0 0 0-.15-.35z"/>
         </g>
-        <g id="attendance">
+        <g id="attendance" mirror-rtl="true">
           <path d="M17 1H1a1 1 0 0 0-1 1v14a1 1 0 0 0 1 1h16a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1zm-6 4h4a1 1 0 0 1 0 2h-4a1 1 0 0 1 0-2zm0 4h4a1 1 0 0 1 0 2h-4a1 1 0 0 1 0-2zM3 5a2 2 0 0 1 4 0v1a2 2 0 0 1-4 0V5zm0 9.99A.994.994 0 0 1 2 14v-2a3 3 0 0 1 6 0v2a.994.994 0 0 1-1 .99L5 15zm8 .01a1 1 0 0 1 0-2h4a1 1 0 0 1 0 2h-4z"/>
         </g>
         <g id="binder">
```

Comment:
I think binder should probably be mirrored, unless there's some branding restrictions around that.